nav2_bt_navigator/behavior_trees/navigate_through_poses_w_replanning_and_recovery.xml

```diff
@@ -9,6 +9,7 @@
       <PipelineSequence name="NavigateWithReplanning">
         <ProgressCheckerSelector selected_progress_checker="{selected_progress_checker}" default_progress_checker="progress_checker" topic_name="progress_checker_selector"/>
         <GoalCheckerSelector selected_goal_checker="{selected_goal_checker}" default_goal_checker="general_goal_checker" topic_name="goal_checker_selector"/>
+        <PathHandlerSelector selected_path_handler="{selected_path_handler}" default_path_handler="PathHandler" topic_name="path_handler_selector"/>
```

Comment:
Add to other XMLs with the selector plugins like the default nav2pose?

Reply:
There are others, but this is the other 'main' one I think you should add it to (also goal / progress checker) https://github.com/ros-navigation/navigation2/blob/main/nav2_bt_navigator/behavior_trees/navigate_to_pose_w_replanning_and_recovery.xml#L11-L12